05-Jason-Luo/wk01/4-thu/js-homework-5.js

```diff
@@ -0,0 +1,35 @@
+// Write a function called cashRegister that takes a shopping cart object. The object contains item names and prices (itemName: itemPrice). The function should return the total price of the shopping cart. 
+
+// Input
+const cartForParty = {
+    banana: "1.25",
+    handkerchief: ".99",
+    Tshirt: "25.01",
+    apple: "0.60",
+    nalgene: "10.34",
+    proteinShake: "22.36"
+};
+
+
+
+function cashRegister(cartForParty) {
+    let items = Object.keys(cartForParty); //converts back to normal array form
```

Comment:
```suggestion
    let items = Object.values(cartForParty); //converts back to normal array form
```
I think its good that you used the `.keys()` method because it helped you get used to handling objects, but just a heads up a shorter way to approach this problem is through the use of the `.values()` method.